<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_pBtnCheckConfig">
         <property name="text">
          <string>查看当前电脑配置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtnBeautifyMouse">
         <property name="text">
          <string>更换鼠标图标</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtnQSSHelper">
         <property name="text">
          <string>QSSHelper</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtnOpenSudoku">
         <property name="text">
          <string>sudoku</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
